src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -3322,28 +3322,34 @@ internal EffectiveParameters(ImmutableArray<TypeWithAnnotations> types, Immutabl
             return paramRefKind;
         }
 
-        // In method group conversions, 'in' is allowed to match 'ref' and 'ref readonly' is allowed to match 'ref' or 'in'.
-        internal static bool AreRefsCompatibleForMethodConversion(RefKind x, RefKind y, CSharpCompilation compilation)
+        // In method group conversions,
+        // - 'in' is allowed to be converted to 'ref',
+        // - 'ref readonly' is allowed to be converted to 'ref' or 'in',
+        // - 'in' is allowed to be converted to 'ref readonly'.
+        internal static bool AreRefsCompatibleForMethodConversion(RefKind source, RefKind destination, CSharpCompilation compilation)
```

Comment:
>source [](http://example.com/codeflow?start=74&length=6)

As mentioned earlier, I find the naming of parameters confusing. Would it make sense to switch to terms like "delegateParameterRefKind" and "candidateMethodParameterRefKind"?